webdriver/tests/bidi/browsing_context/reload/reload.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_payload(bidi_session, inline, new_tab):
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"],
+                                                 url=url)
+
+    result = await bidi_session.browsing_context.reload(
+        context=new_tab["context"])
+    assert result == {}
+
+
+async def test_relative_url(bidi_session, new_tab, url):
+    url = url(
+        "/webdriver/tests/bidi/browsing_context/navigate/support/empty.html")
```

Comment:
Hm, what is this test trying to achieve? The page that we are loading here is not relative given that the [`url` fixture](https://github.com/web-platform-tests/wpt/blob/92af94020c76c6f5d03306e9b3fe74409244ea12/webdriver/tests/support/fixtures.py#L202-L210) is using `urlunsplit` and as such constructs an absolute URL.

I assume we can only test that from within an iframe which has a relative URL to that top-level page?

Reply:
This was mostly mirrored from wpt/webdriver/tests/bidi/browsing_context/navigate/navigate.py.

Removing it then.